services/ui-src/src/changeRequest/DetailView.tsx

```diff
@@ -197,41 +254,17 @@ const DetailSection = ({
       </section>
       <div className="read-only-submission">
         <section className="detail-section">
-          <h2>Package Details</h2>
-          {detail.waiverAuthority && (
-            <Review heading="Waiver Authority">
-              {AUTHORITY_LABELS[detail.waiverAuthority] ??
-                detail.waiverAuthority}
-            </Review>
-          )}
-          {detail.componentId && (
-            <Review heading={PAGE_detail[detail.componentType].idLabel}>
-              {detail.componentId}
-            </Review>
-          )}
-          {detail.componentType && (
-            <Review heading="Type">
-              {PAGE_detail[detail.componentType].detailHeader}
-            </Review>
-          )}
-          {detail.territory && (
-            <Review heading="State">{territoryMap[detail.territory]}</Review>
-          )}
-          {detail.submissionTimestamp && (
-            <Review heading="Date Submitted">
-              {formatDetailViewDate(detail.submissionTimestamp)}
-            </Review>
-          )}
-          {ChangeRequest.MY_PACKAGE_GROUP[detail.componentType] ===
-            ChangeRequest.PACKAGE_GROUP.WAIVER && (
-            <Review heading="Proposed Effective Date">
-              {detail.proposedEffectiveTimestamp
-                ? formatDetailViewDate(detail.proposedEffectiveTimestamp)
-                : "N/A"}
-            </Review>
+          <h2>{pageConfig.detailHeader} Details</h2>
+          {pageConfig.detailsSection?.map(
+            (item, index) =>
+              (detail[item.fieldName] || item.default) && (
+                <Review key={index} heading={item.heading}>
+                  {detail[item.fieldName] ?? item.default}
+                </Review>
+              )
           )}
         </section>
-        <section className="detail-section">
+        <section className="detail-section ds-u-margin-bottom--7">
           <FileList
             heading="Base Supporting Documentation"
```

Comment:
This should say "Supporting Documentation" per the Figma. Not sure how HCD wants it to appear for other component types.